examples/validation-app/src/__tests__/acceptance/validate.acceptance.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-file-upload-download
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Client} from '@loopback/testlab';
+import {ValidationApplication} from '../..';
+import {setupApplication} from './test-helper';
+
+const validCoffeeShop = {
+  city: 'Toronto',
+  phoneNum: '416-111-1111',
+  capacity: 10,
+};
+
+describe('validate properties', () => {
+  let client: Client;
+  let app: ValidationApplication;
+
+  before(givenAClient);
+
+  after(async () => {
+    await app.stop();
+  });
+
+  async function givenAClient() {
+    ({app, client} = await setupApplication());
+  }
+
+  it('can create CoffeeShop with valid properties', async () => {
+    await client
+      .post('/coffee-shops')
+      .send(validCoffeeShop)
+      .expect(200);
+  });
+
+  it('create CoffeeShop failed with city name exceeds length limit', async () => {
+    const invalidCityNameCS = validCoffeeShop;
+    invalidCityNameCS.city = 'Tooooooooooronto';
+    await client
+      .post('/coffee-shops')
+      .send(invalidCityNameCS)
+      .expect(422);
```

Comment:
Does the error message say what the issue is? If I was a user, I'd want to know what was wrong

Reply:
the error says what the issue is. but the test code is not checking.